static/templates/stream_settings/stream_types.hbs

```diff
@@ -1,3 +1,15 @@
+{{#if ask_to_announce_stream}}
+    <div id="announce-new-stream">
+        <label class="checkbox">
+            <input type="checkbox" name="announce" value="announce" checked />
+            <span></span>
+            {{t "Announce new stream in" }}<strong> {{{stream_privacy_symbol_html}}}{{notifications_stream}}</strong>
```

Comment:
Don't think this is the correct way to tag this for translation. You can refer to `static/templates/confirm_dialog/confirm_deactivate_stream` to see how it should be done.

Reply:
I tried changing it into sth similar to static/templates/confirm_dialog/confirm_deactivate_stream. Everything works as expected but it throws error in console which is given below.
<img width="440" alt="Screen Shot 2022-11-19 at 7 57 06 PM" src="https://user-images.githubusercontent.com/53159963/202877851-f26d3e23-709c-4d9f-a896-c345fc4ef456.png">

### announce_stream.hbs
```
{{#tr}}
    Announce new stream in <strong>{{{stream_privacy_symbol_html}}}{{notifications_stream}}</strong>.
{{/tr}}
```

### stream_types.hbs
```
{{#if ask_to_announce_stream}}
    <div id="announce-new-stream">
        <label class="checkbox">
            <input type="checkbox" name="announce" value="announce" checked />
            <span></span>
        {{> announce_stream
            stream_privacy_symbol_html=stream_privacy_symbol_html
            notifications_stream=notifications_stream }}
    </div>
    {{> ../help_link_widget link="/help/configure-notification-bot#new-stream-announcements"}}
</label>
{{/if}}
```

Reply:
Also `Announce new stream in` is the only part which needs translation. So, I don't know if translation is really an issue in the above PR.

Reply:
@sahil839 , Please check this.

Reply:
See https://zulip.readthedocs.io/en/latest/translating/internationalization.html#key-details-about-human-language, which mentions that word order may vary in different languages, so we need to make sure that the sentence as a whole is tagged for translation.

Reply:
You cannot use `strong` tag like this inside `#tr`. `confirm_deactivate_stream.hbs` uses `inline z-stream` and you would need to do something similar. See last part in https://zulip.readthedocs.io/en/latest/translating/internationalization.html#handlebars-templates